src/my_project/tagging/image_scoring.py

```
@@ -0,0 +1,149 @@
+from my_project.classes.tagging_classes import Candidate
+from my_project.tagging.requester import HttpRequest
+
+logger = logging.getLogger(__name__)
+
+
+class ImageScorer:
+    HASH_SIZE = 16
+    WEIGHTS = {"phash": 0.7, "dhash": 0.2, "ahash": 0.1}
+    FINAL_WEIGHTS = {"hash": 0.6, "hue_hist": 0.4}
+
+    def __init__(self, actual_cover: BytesIO, http: HttpRequest):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)